.travis.yml

```diff
@@ -83,6 +83,7 @@ script:
   - |
     if [[ "$E2E" == "1" ]]; then
       npm run build:test || exit 1 # Build for tests.
+      docker run --rm -v "$PWD:/app" -v "$HOME/.cache/composer:/tmp/cache" composer install
```

Comment:
@eugene-manuilov Why was this added when the same command is already run for E2E tests above on line `47` in the `before_script`?

Reply:
We need to regenerate autoload map to include Manifest.php file generated after `npm run build:test`. Probably not the best approach to do it, but it works.

Reply:
Yeah, worst case we can run `composer run autoload-includes` to only regenerate the Site Kit includes without needing to run all of the prefixing again, but I think we can do better than that.